warehouse/routes.py

```diff
@@ -87,11 +87,36 @@ def includeme(config):
         "/account/register/",
         domain=warehouse,
     )
+
+    # Management (views for logged-in users)
+    config.add_route("manage.profile", "/manage/profile/", domain=warehouse)
+    config.add_route("manage.projects", "/manage/projects/", domain=warehouse)
     config.add_route(
-        "accounts.edit_gravatar",
-        "/user/{username}/edit_gravatar/",
-        factory="warehouse.accounts.models:UserFactory",
-        traverse="/{username}",
+        "manage.project.settings",
+        "/project/{name}/settings/",
```

Comment:
should this and the following three routes be namespaces under `/manage`?

it feels valid to me to keep all "edit" activity under a single top level path.

perceived advantages:

  - easier to spot/search in logs
  - easier to manage routing/caching/etc in the layers above warehouse